### PR TITLE
Chalice - passing params to rook (RK-2074)

### PR DESCRIPTION
**Pull request by BenTheKlein**

### CODE REVIEW

File: python-aws-chalice/Readme.md [hunk 1]
```diff
@@ -24,4 +24,11 @@ from rook.serverless import RookoutChalice
 app = RookoutChalice(app_name='python-aws-chalice')
 ```
 
+You can also specify rook params using the api (use before the RookoutChalice initialization):
```
Comment (Rookeret73) [inline python-aws-chalice/Readme.md]: You may also pass parameters using the Rookout API:

File: python-aws-chalice/app.py [hunk 2]
```diff
@@ -1,5 +1,9 @@
 from rook.serverless import RookoutChalice
 
+# In order to pass params to the rook directly - uncomment the section below
```
Comment (Rookeret73) [inline python-aws-chalice/app.py]: In order to pass parameters using the Rookout API, uncomment the section below:

**Comment by BenTheKlein** [comment]
FIX TEST FIRST

**Comment by BenTheKlein** [comment]
@Rookeret73 please re-approve it :)

**Pull request merged**